addons/web/static/src/views/fields/input_field_hook.js

```diff
@@ -122,7 +122,7 @@ export function useInputField(params) {
 
     useBus(env.bus, "RELATIONAL_MODEL:WILL_SAVE_URGENTLY", () => commitChanges(true));
     useBus(env.bus, "RELATIONAL_MODEL:NEED_LOCAL_CHANGES", (ev) =>
-        ev.detail.proms.push(commitChanges())
+        ev.detail.proms.push(commitChanges(false))
```

Comment:
could you write a commit message. FGE think it should be good.

Reply:
In fact, it is just a detail that I saw when I create my hook, the parameter is not passed to the method but there is no issue since the parameter will be falsy (= `undefined`). I cannot use that hook since we assume the `record` is required in the props of the component. 😢

If you prefer doing that change to be sure we pass the parameter then okay, I will do the commit with a correct message. 🙂

Reply:
no need, just remove that part then and keep the othe part of the commit with the export

Reply:
Okay, thanks 🙂